app/src/sharedTest/java/org/oppia/android/app/onboarding/OnboardingFragmentTest.kt

```diff
@@ -223,9 +223,19 @@ class OnboardingFragmentTest {
     }
   }
 
+  @Test
+  fun testNextButtonSpotlight_setToShowOnStartup_checkSpotlightIsShown() {
```

Comment:
What about making sure it doesn't reshow? That seems like a worthwhile test to include.

Reply:
This should be shown each time the app is started without being onboarded simce it isnt tied to a profile